charts/index.yaml

```diff
@@ -45,8 +45,18 @@ entries:
   cromwell:
   - apiVersion: v2
     appVersion: 68-8e12ab5
-    created: "2021-10-29T16:37:08.32811-04:00"
-    description: Simple Cromwell deployment with Postgres database running jobs locally
+    created: "2021-11-02T16:17:45.942353-04:00"
+    description: Simple Cromwell deployment with in-memory database running jobs locally
```

Comment:
This is now Postgres database instead of in-memory.

Reply:
Hmmm maybe I missed a step. This file was updated by running `helm repo index --url https://broadinstitute.github.io/cromwhelm/charts/ charts`. Do I need to change something else?

Reply:
Oh, I bet it's picking up that description from somewhere else then. I edited it here, but that probably was wrong. Don't worry about it for now… it's not a big deal.

Reply:
I see… it should be updated in Chart.yaml (and that's where the Cromwell version is also).

Reply:
Thank you! I will update in Chart.yaml and regenerate the files